packages/rs-drive-abci/src/execution/platform_events/state_transition_processing/validate_fees_of_event/v0/mod.rs

```diff
@@ -60,6 +64,13 @@ where
                     )
                     .map_err(Error::Drive)?;
 
+                ExecutionOperation::add_many_to_fee_result(
```

Comment:
Did you consider to add this method to FeeResult?

Reply:
We can not, as FeeResult is in DPP